lib/solana/rpc/request.ex

```diff
@@ -79,13 +79,25 @@ defmodule Solana.RPC.Request do
     {"getBlock", [start_slot, encode_opts(opts)]}
   end
 
+  @doc """
+  Returns the latest blockhash.
+
+  For more information, see [the Solana
+  docs](https://solana.com/hi/docs/rpc/http/getlatestblockhash).
```

Comment:
Please reference the English documentation instead: https://solana.com/docs/rpc/http/getlatestblockhash